### ISSUE TITLE
Flame Shock Debuff on Mobs bugged

### ISSUE THREAD

**Issue by AranMat** [WoWManiaUK/Redemption#4517]
![Capture](https://user-images.githubusercontent.com/38553469/80819638-9dcd6500-8bd5-11ea-9255-2ba004ed70b9.JPG)
**Links:**

**What is Happening:**
Flame shock display icon on Boss is normal, but once lava burst is applied, a visual bug shows that it looks like it has 1 second remaining , meanwhile still over 10 seconds to go.

**What Should happen:**
Flame shoch icon should gradualt turn grey as timer reduces

**Comment by Rushor** [comment]
maybe client issue?

**Comment by wow-mania** [comment]
Visual only.

**Comment by jeyrella** [comment]
Whilst it is visual, this does have some raiding gameplay implications for the class.

Flame shock is a dot ele shamans have to keep up on enemies since it makes their lava burst crit 100% of the time. The 4t10 bonus makes it so that your lava burst against a target affected by flame shock will extend the duration by 2 ticks.

Problem is, if you just flame shocked and cast lava burst, there's gonna be 10+ seconds left on the debuff, but the indicator will bug and go to 0s left. This makes it very hard for Elemental Shamans to judge when to refresh flame shock without clipping it and accidentally casting lava burst after it expires, leading to a considerable dps loss for the class.

Basically it makes the ele shaman rotation awkward for no reason.  Yes it is only visual, but is annoying.

Druid's moonfire dot behaves correctly when extended by starfire while using the glyph. Might be able to copy the behavior over.

**Comment by Keader** [comment]
try now

**Comment by jeyrella** [comment]
Looks good
https://youtu.be/rEA24lf3prk
